File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { YandereApiV2SDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await YandereApiV2SDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- YandereApiV2 SDK exists test

local sdk = require("yandere-api-v2_sdk")

describe("YandereApiV2SDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
